actors/index-actor.ts

```diff
@@ -0,0 +1,39 @@
+export class IndexActor {
+  readonly page: Page;
+  readonly mainPage: MainPage;
+  readonly productSelectionPage: ProductSelectionPage;
+
+  constructor(page: Page, mainPage: MainPage, productSelectionPage: ProductSelectionPage) {
+    this.page = page;
+    this.mainPage = mainPage;
+    this.productSelectionPage = ProductSelectionPage; 
+  }
+
+  async goto() {
+    await this.page.goto('/cockpit/@localhost/agama/index.html');
+  }
+
+  async handleProductSelectionIfAny() {
+    // possible first screens the user can find
```

Comment:
do not use "screens" like in Installer, in Agama let's use pages, as we are dealing with web pages.

Reply:
Updated